pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
bah - my mistakes.  I intentionally updated the docs so that they didn't point to old images but forgot about the implications of the tag hash.

I agree that keeping the `README` in `./docs` would be preferred